src/games/supported/LostLuggage.cpp

```diff
@@ -61,24 +61,6 @@ bool LostLuggageSettings::isMinimal(const Action& a) const {
   }
 }
 
-bool LostLuggageSettings::isLegal(const Action& a) const {
```

Comment:
I don't think this should be removed -- it will reset the game mid-way through.

Reply:
I see. I'm removing this commit from the pull request. We should discuss offline.